draft-ietf-quic-transport.md

```diff
@@ -542,68 +545,132 @@ the server supports, as shown below.
 See {{version-negotiation}} for a description of the version negotiation
 process.
 
+
 ## Cleartext Packets {#cleartext-packet}
 
-Cleartext packets are sent during the handshake prior to key negotiation. A
-Client Cleartext packet contains:
+Cleartext packets are sent during the handshake prior to key negotiation.
 
-* Octet 0: 0x82
-* Octets 1-8: Connection ID (initial)
-* Octets 9-12: Packet number
-* Octets 13-16: Version
-* Octets 17+: Payload
+Cleartext packets all contain the current QUIC version in the version field.
 
-Non-Final Server Cleartext packets contain:
+The packet number is randomized for the Client Initial packet
+({{packet-client-initial}} and the first Server Cleartext packet
+({{packet-server-cleartext}}).  It increases by one for each packet sent
+thereafter, see {{packet-numbers}} for details.  The Server Stateless Retry
+packet ({{packet-server-stateless}}) contains the packet number of the packet
+that it is responding to.
 
-* Octet 0: 0x83
-* Octets 1-8: Connection ID (echoed)
-* Octets 9-12: Packet Number
-* Octets 13-16: Version
-* Octets 17+: Payload
+The payload of cleartext packets also includes an integrity check, which is
+described in {{QUIC-TLS}}.
 
-Final Server Cleartext packets contains:
 
-* Octet 0: 0x84
-* Octets 1-8: Connection ID (final)
-* Octets 9-12: Packet Number
-* Octets 13-16: Version
-* Octets 17+: Payload
+### Client Initial Packet {#packet-client-initial}
 
-The client MUST choose a random 64-bit value and use it as the initial
-Connection ID in all packets until the server replies with the final Connection
-ID. The server echoes the client's Connection ID in Non-Final Server Cleartext
-packets.  The first Final Server Cleartext and all subsequent packets MUST use
-the final Connection ID, as described in {{connection-id}}.
+The Client Initial packet has a type octet of 0x82.  It carries the first
+cryptographic handshake message sent by the client, prior to receiving any
+packets from the server.
 
-The payload of a Cleartext packet consists of a sequence of frames, as described
-in {{frames}}.  Cleartext packets also include an integrity check, described in
-{{QUIC-TLS}}.
+The client populates the connection ID and packet number fields with randomly
+selected values.  The randomized packet number is incremented for each
```

Comment:
Just "packet number" -- If the packet number is incremented for each subsequent packet, it's not randomized.